package.json

```diff
@@ -144,7 +148,16 @@
 	"nyc": {
 		"report-dir": "coverage/cypress",
 		"reporter": [
-			"html"
+			"html",
+			"json",
+			"lcov"
+		],
+		"include": ["src/"],
+		"exclude": [
+			"cypress/**/*.js",
+			"jest-test-setup.js",
```

Comment:
Safe to remove lines 158 & 159 here, since the istanbul ignore directive has been added to those files.